src/ui-kit/components/Avatar/Avatar.styled.jsx

```diff
@@ -0,0 +1,30 @@
+import styled from '@emotion/styled';
+import { Avatar } from '@mui/material';
+
+const sizeController = (size = 'xl') => {
```

Comment:
I was showing you an example, not him. And why such dimensions, figma has completely different ones